### PR TITLE
Move CI to Github Actions

### PR DESCRIPTION
**Pull request by gaborbernat**
Signed-off-by: Bernát Gábor <gaborjbernat@gmail.com>

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          import codecs
+          import os
+          import platform
+          import sys
+          cpy = platform.python_implementation() == "CPython"
+          base =("{}{}{}" if cpy else "{}{}").format("py" if cpy else "pypy", *sys.version_info[0:2])
+          env = "TOXENV={}\n".format(base)
+          print("Picked:\n{}for{}".format(env, sys.version))
+          with codecs.open(os.environ["GITHUB_ENV"], "a", "utf-8") as file_handler:
+               file_handler.write(env)
```
Comment (asottile) [inline .github/workflows/check.yml:59]: this can probably just run `tox -e py` maybe?

**Review by asottile — APPROVED** [review]

File: .github/workflows/check.yml (hunk 1 above)
Comment (gaborbernat) [reply .github/workflows/check.yml:59]: That would not work for two reasons:

- we're currently always using tox with 3.10 to run the test suite of the tox tool, and -e py would mean we'd always test under 3.10.
- we'd have to massage the coverage filenames to differentiate between runs, as uploading the same artefact twice operates in overwrite mode.

Comment (asottile) [reply .github/workflows/check.yml:59]: ahh makes sense

**Pull request merged**